src/main/scala-2.12/com/linagora/gatling/imap/protocol/ImapSessions.scala

```diff
@@ -27,6 +27,13 @@ class ImapSessions(protocol: ImapProtocol) extends BaseActor {
 
   override def receive: Receive = {
     case cmd: Command =>
+      logger.trace(
```

Comment:
I don't like multiline logs because they are not grep-able. Can we use MDC with actors ? In that case, you can have a multiline pattern in your logback configuration if you want

Reply:
or simpler we can even remove the log now. As we can't run with them in the ci, because with it the log is too long and the ci stop for this reason.